x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -150,19 +153,20 @@ export const ModelsList: FC<Props> = ({
                 type: [
                   model.model_type,
                   ...Object.keys(model.inference_config),
-                  ...(isBuiltInModel(model) ? [BUILT_IN_MODEL_TYPE] : []),
+                  ...(isBuiltInModel(model as ModelItem) ? [BUILT_IN_MODEL_TYPE] : []),
                 ],
               }
             : {}),
-        };
+        } as ModelItem;
         newItems.push(tableItem);
 
         if (itemIdToExpandedRowMap[model.model_id]) {
           expandedItemsToRefresh.push(tableItem);
         }
       }
 
-      // Need to fetch state for all models to enable/disable actions
+      // Need to fetch stats for all models to enable/disable actions
+      // TODO combine fetching models definitions and stats into a single function
```

Comment:
Is this todo still needed for this PR? Or is it meant to be addressed in a follow up?

Reply:
this one is for the follow-up, perhaps for 8.9